### PR TITLE
Adds a toggle for the announcer voice.

### PR DESCRIPTION
**Pull request by printer16**
Adds a toggle for the announcer voice as some people find them annoying. Related thread: https://forums.aurorastation.org/viewtopic.php?f=18&t=10212 

Haven't tested it yet.

### CODE REVIEW

**Comment by printer16** [comment]
Moved it.

File: code/__defines/misc.dm [hunk 1]
```diff
@@ -75,15 +75,15 @@ Will print: "/mob/living/carbon/human/death" (you can optionally embed it in a s
...
 #define SAFETY_CHECK 0x10
 
-
 #define TOGGLES_DEFAULT (SOUND_ADMINHELP|SOUND_MIDI|SOUND_AMBIENCE|SOUND_LOBBY|CHAT_OOC|CHAT_DEAD|CHAT_GHOSTEARS|CHAT_GHOSTSIGHT|CHAT_PRAYER|CHAT_RADIO|CHAT_ATTACKLOGS|CHAT_LOOC)
 
 //Sound effects toggles
 #define ASFX_AMBIENCE	1
 #define ASFX_FOOTSTEPS	2
 #define ASFX_VOTE		4
+#define ASFX_VOX		6
```
Comment (skull132) [inline code/__defines/misc.dm]: This should be. `8`.

**Comment by skull132** [comment]
Changelog req still. Otherwise okay.

**Comment by printer16** [comment]
Added the log.

**Pull request merged**